Nighttime-Operations/Auxiliary-Telescope/AT-Shutdown/Shutdown.rst

```diff
@@ -8,10 +8,117 @@
 Shutdown
 #######################
 
-Under construction.
+Stopping Scheduler
+^^^^^^^^^^^^^^^^^^
+
+The Scheduler automatically adds new targets to the queue until the start of nautical morning twilight.
+If there are no targets, the Scheduler will stop adding new ones.
+If you need to stop the Scheduler manually, you can use the ``auxtel/scheduler/stop.py`` script from ScriptQueue without any additional configurations.
+
+See more detailed descriptions on `Scheduler <https://obs-ops.lsst.io/Nighttime-Operations/Scheduler/night-time-operations/index.html>`__.
+
+
+Shutdown AuxTel
+^^^^^^^^^^^^^^^
+
+After stopping scheduler, you can shutdown AuxTel using ``axutel/shutdown.py`` script from ScriptQueue. 
+``shutdown.py`` script does not any configurations. This script includes:
+
+**1. Closing the M1 cover.**
+: First, disable ATAOS corrections. Then, close the M1 cover and vent gates.  
+If mirror is opened, ``shutdown`` script will not able to close the dome slit. 
+
+**2. Closing the Dome Shutter.**
+: Closing the dome shutter and dropout door if it is opened.  
+ 
+**3. Parking the Dome** 
+: The parking position is az=285.0 deg. Then, the dome following is disable after parking. 
+
+**4. Parking the Telescope**
+: The parking position is El = 80 deg, Az = 0.0 deg, rot = 0.0 deg for telescope.
+Tracking will then be stopped.  
+
+**5. Putting CSCs in Stanby.**
+: The AT CSC components ``ATMCS``, ``ATOS``, ``ATPtg``, ``ATDome``, ``ATDomeTrajector``, ``ATOS``, ``ATPneumatics``, ``ATHexapod``  will be change to Stanby. 
+
+After parking and shutdown for Auxillary Telescope are completed, you can check the status of Dome and Telescope on the LOVE. 
+
+- **Status of Telescope and Dome** 
+.. image:: ./_static/ATDome_park.png
+    :name: Auxiliary Telescope and Dome at the parking position
+
+- **Status of Mirrors and Mirror Covers**
+.. image:: ./_static/AT_LightPath.png
+   :scale: 50 %
+
+   
```

Comment:
I think this part also can go under Post condition because is something that happens after shutdown and park, right?

Reply:
Post condition as well.